spec/spec_helper.rb

```diff
@@ -1,6 +1,8 @@
 require "set"
 require "objspace"
 
+at_exit { GC.start }
```

Comment:
Is this necessary?

Reply:
To clarify, if that's needed, shouldn't `ruby_memcheck` provide a `ruby_memcheck/spec_helper` or something like that to do it automatically?

Reply:
Yes this is needed to run a GC at shutdown to clean up any leftover objects to reduce false positives. It's not in a file that is `require`'d because I don't anticipate ever needing to do anything other than this `GC.start`. ruby_memcheck is designed to be as transparent as possible, so I don't plan on extending or modifying RSpec/minitest in any way.

Reply:
Or even better, since you have control over the command: https://github.com/Shopify/ruby_memcheck/blob/a665f906ea5e49fbf4baffd2ae1d6364e924691e/lib/ruby_memcheck/rspec/rake_task.rb#L42 you could pass a `RUBYOPT` to have this code loaded automatically.